src/api/migrations/versions/6b603528dac8_add_system_profile.py

```
@@ -0,0 +1,121 @@
+        "sys_user_input",
+        "sys_user_language",
+    ]
+    with app.app_context():
+        for profile_key in profile_keys:
+            profile_item = ProfileItem.query.filter_by(profile_key=profile_key).first()
+            if profile_item:
+                profile_item.status = 0
+                profile_item_i18n = ProfileItemI18n.query.filter_by(
+                    parent_id=profile_item.id
+                ).all()
+                for profile_item_i18n in profile_item_i18n:
```

Ruff (0.14.3)

Loop control variable `profile_item_i18n` overrides iterable it iterates

(B020)